contracts/VToken.sol

```diff
@@ -662,7 +662,7 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
 
         /* We emit a Transfer event, and a Redeem event */
         emit Transfer(redeemer, address(this), redeemTokens);
-        emit Redeem(redeemer, redeemAmount, redeemTokens);
+        emit Redeem(redeemer, redeemAmount, redeemTokens, accountTokens[redeemer]);
```

Comment:
Reading from storage uses alot of gas. Would be more gas efficient if we do something like:
```
uint256 balanceAfter = accountTokens[redeemer] - redeemTokens;
accountTokens[redeemer] = balanceAfter;
emit Redeem(redeemer, redeemAmount, redeemTokens, balanceAfter);
```